.gitleaks.toml

```diff
@@ -0,0 +1,88 @@
+# Global allowlist
+[allowlist]
+paths = [
+    '''node_modules''',
+    '''.git'''
+]
+
+# Custom rules
+[[rules]]
+id = "alchemy-api-key"
+description = "Alchemy API Key"
+regex = '''https://[a-zA-Z0-9-]+\.g\.alchemy\.com/v2/[a-zA-Z0-9_-]+'''
+keywords = [
+    "alchemy",
+    "unichain-mainnet"
```

Comment:
does this mean custom domains will not match?

Reply:
ah no it's just keywords it'll look for to flag up things earlier for search

can remove it

Reply:
```suggestion
    "alchemy.com"
```